test/integration_test/basic_test.go

```diff
@@ -208,3 +221,70 @@ func BasicUpgrade(tc *TestCase) func(*testing.T) {
 		require.NoError(t, err)
 	}
 }
+
+func InstallWithTelemetry(tc *TestCase) func(*testing.T) {
+	return func(t *testing.T) {
+		if tc.ShouldSkip() {
+			t.Skip()
+		}
+
+		testSpec := tc.TestSpec(t)
+		cluster, ok := testSpec.(*op_corev1.StorageCluster)
+		require.True(t, ok)
+
```

Comment:
you could parse and create the secret here, then just call `BasicInstall(tc)(t)`

Reply:
Let's still keep it separate so we can test separately after making changes.